mysql-test/suite/rpl/t/rpl_xa_prepare_gtid_fail.test

```diff
@@ -25,9 +25,9 @@ source include/have_debug_sync.inc;
 --connection slave
 --source include/stop_slave.inc
 
---let $save_par_thds= `SELECT @@global.slave_parallel_threads`
---let $save_strict_mode= `SELECT @@global.gtid_strict_mode`
---let $save_innodb_lock_wait_timeout= `SELECT @@global.innodb_lock_wait_timeout`
+set @save_par_thds= @@global.slave_parallel_threads;
```

Comment:
As a remark, I always thought of a macro which would assign to a new value while save in background into magic "save_" list of items so that the eventual `rpl_end.inc` would restrore from them.